js/components/cohort-definitions/cohort-definition-manager.html

```diff
@@ -112,8 +112,9 @@
 
 						<div class="clear"></div>
 						<div class="cohort-conceptset-button-pane pull-right">
-						  <button type="button" class="btn btn-sm btn-danger" data-bind="click:function() { $component.deleteConceptSet()};">Delete Concept Set</button>
-						  <button type="button" class="btn btn-sm btn-primary" data-bind="click: function() { $component.closeConceptSet()};">Close Concept Set</button>
+						  <button type="button" class="btn btn-sm btn-danger" data-bind="click:function() { $component.deleteConceptSet(); }">Delete Concept Set</button>
+						  <button type="button" class="btn btn-sm btn-success" data-bind="click:function(){ $component.showSaveConceptSet(); }, enable: canCreateConceptSet">Save Concept Set</button>
```

Comment:
Instead of the label 'Save Concept Set', I'd prefer if this were named "Copy To Concept Set Repository".